<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

<title>Igor  Ausiany | NCN Preludium</title>
<meta name="description" content="Teaching portfolio.
">

<!-- Open Graph -->


<!-- Bootstrap & MDB -->
<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

<!-- Fonts & Icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"  integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">


<!-- Code Syntax Highlighting -->
<link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />

<!-- Styles -->
<link rel="shortcut icon" href="/~kgogolewski/igorteacher/assets/img/favicon.ico">
<link rel="stylesheet" href="/~kgogolewski/igorteacher/assets/css/main.css">

<link rel="canonical" href="/~kgogolewski/igorteacher/projects/1_project/">

<!-- Theming-->




    
<!-- MathJax -->
<script type="text/javascript">
  window.MathJax = {
    tex: {
      tags: 'ams'
    }
  };
</script>
<script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.1.2/es5/tex-mml-chtml.js"></script>
<script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>


  </head>

  <body class="fixed-top-nav ">

    <!-- Header -->

    <header>

    <!-- Nav Bar -->
    <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
    <div class="container">
      
      <a class="navbar-brand title font-weight-lighter" href="/~kgogolewski/igorteacher/">
       <span class="font-weight-bold">Igor</span>   Ausiany 
        
        <a class="support-button" href="https://zrzutka.pl/en/ghr8d9/pay">
            <i class="fas fa-piggy-bank"></i> Support Me
        </a>

       
      </a>
      
      <!-- Navbar Toogle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <!-- About -->
          <li class="nav-item ">
            <a class="nav-link" href="/~kgogolewski/igorteacher/">
              about
              
            </a>
          </li>
          
          <!-- Other pages -->
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/~kgogolewski/igorteacher/certificates/">
                certificates
                
              </a>
          </li>
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/~kgogolewski/igorteacher/teaching/">
                teaching
                
              </a>
          </li>
          
          
          
        </ul>
      </div>
    </div>
  </nav>

</header>


    <!-- Content -->

    <div class="container mt-5">
      <div class="post">

  <header class="post-header">
    <h2 class="post-title">NCN Preludium</h1>
    <p class="post-description">Bayesian analysis of bladder cancer subtypes based on high-throughput data</p>
  </header>

  <article>
    <h2 id="research-project-objectives">research project objectives</h2>
<p>Technological progress, present in almost every science field, is tightly related to gathering the huge amounts of data. 
The key example here is molecular biology, which thanks to automated diagnostics can provide a description of given patient with gigabytes of data from high throughput technologies. 
One of such technologies are microarray methods for gene expression measurements or spectrometric determination of metabolites concentrations in cells that. 
Nevertheless, whatever the type of technology is, provided data sets carry important and useful information that can and should be retrieved and used in diagnostics, personalised medicine or drug design procedures.</p>

<p>On the other hand, an ongoing, joint work of molecular biologists, bio-physicists and -chemists results in systematic description of processes that occur in human organisms and reveals possibilities for highly specific inference. 
A good example of a molecular mechanism description are signalling pathways, that with their formal representation can be deeply studied to better understand the complex system of a human cell.</p>

<p>This project aimed to derive an integrative model that analyse series of high throughput experimental data in two interrelated contexts:</p>
<ul>
  <li>locally - the independent analysis of individual signalling pathways, that will allow to select the most significantly (in a statistical sense) perturbed signalling cascades in inter-cellular communication;</li>
  <li>globally - the analysis of a full metabolic network, that describes all possible sources of phenotypic perturbations (with respect to the homeostatic profile) in given experimental data.</li>
</ul>

<h2 id="project-results">project results</h2>

<h4 id="mph---molecular-process-heterogeneity">MPH - Molecular Process Heterogeneity</h4>
<div class="row justify-content-sm-center">
   <div class="col-sm-4 mt-3 mt-md-0">
        <img class="img-fluid rounded z-depth-1" src="/~kgogolewski/igorteacher/assets/img/MPHworkflow.jpg" alt="" title="MPH computational worflow" />
   </div>
   <div class="col-sm-4 mt-3 mt-md-0">
        <img class="img-fluid rounded z-depth-1" src="/~kgogolewski/igorteacher/assets/img/MPHexperiment.jpg" alt="" title="MPH experimental pipeline" />
   </div>
 
</div>
<div class="caption">
During the realization of the project two worflows were described and developed. 
The first one is MPH method allows for the analysis of the activity of molecular processes on the basis of transcriptomic data in homogeneous cell populations. 
</div>

<p>The method carries out two phases of non-negative matrix factorization, resulting in an estimate of the functional composition of a homogeneous population of cells along with potential marker genes and activity patterns of specific molecular processes. 
In the first step, we adapt the Beyesian DSection method previously proposed for transcriptome analysis of heterogeneous populations. 
Using the set a priori knowledge regarding the expected proportion of functional sub-populations, the first stage estimates the transcriptomic profiles characterizing the activity of the expected molecular processes. 
Then, the obtained profiles are used to extract the marker genes of individual processes and use them in the second step based on the iterative ssKL method based on minimizing the Kullback-Leibler divergence. 
The result of the second step of the method is the percentage share of cells in individual molecular processes, and moreover, for each of the processes, new potential marker genes are indicated.</p>

<h4 id="extended-fba-analysis">extended FBA analysis</h4>

<p>The second proposed method of analyzing transcriptomic data, taking into account metabolomic knowledge, was the processing of transcriptomic data in the context of the metabolic network of a human cell. This method is aimed at determining the profiles of metabolic activity of patients with diagnosed cancer and then at determining the key metabolic reactions that could constitute biomarkers of the relevant molecular subtypes of a given tumor.</p>

<div class="row">
    <div class="col-sm mt-3 mt-md-0">
        <img class="img-fluid rounded z-depth-1" src="/~kgogolewski/igorteacher/assets/img/extendedFBA.jpg" alt="" title="Extended FBA method" />
    </div>
</div>
<div class="caption">
Extended FBA (Flux Balance Analysis) method determines the activity of metabolic reactions on the basis of the integration of transcriptomic knowledge with the metabolic model of the human cell, taking into account the correction for the heterogeneous distribution of enzymes in the metabolic network.
</div>

<p>The first step in this method is the preparation of the patient’s Personalized Genome-Scale Metabolic Model. We solve this task by integrating the individual transcriptomic profile with the general model of the human cell metabolic network RECON 2.2. In general, the design of the model is based on the determination of the initial activity state of the enzymatic reactions in the lattice. The decision on the status of a reaction is based on the activity of specific genes or groups of genes that guarantee the production of enzymes necessary for a given reaction to take place.</p>

<p>Then, on the basis of a given personalized model, a linear problem is formulated, which maximizes the compliance of the reaction activity after taking into account the structure of the metabolic network with the observed profiles of gene expression levels. The linear problem formulated at this stage belongs to the group of FBA (Flux Balance Analysis) problems that are commonly used to determine the level of flows in networks. The solution to the linear programming problem is the patient’s metabolic landscape, which describes the most optimal distribution of the activity of individual metabolic reactions.</p>

<p>The last step in the construction of our method was the introduction of an amendment taking into account the heterogeneous and incomplete knowledge of human metabolism. The presence of enzymes that coordinate (or co-coordinate) the activity of many reactions is the reason for the observation of artifacts related to the structure of the metabolic network at the stage of analyzing the metabolic profiles of patients. Hence, the last step of the method is to correct the estimated activity of the reaction in accordance with the belonging of the reaction to the cell compartment and the frequency of co-ordinating enzymes.</p>

<h4 id="funding">Funding</h4>
<p>The project was funded by Polish National Science Center, Grant no. 2016/21/N/ST6/01507.</p>

<div class="publications">
  <h2>Related publications</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">BioMedRes</abbr>
    
  
  </div>

  <div id="GogolewskiBioMedRes" class="col-sm-8">
    
      <div class="title">Inferring Molecular Processes Heterogeneity from Transcriptional Data</div>
      <div class="author">
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  Wronowska, W.,
                
              
            
          
        
          
          
          
            
              
                
                  Lech, A.,
                
              
            
          
        
          
          
          
            
              
                
                  Lesyng, B.,
                
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Biomed Res Int</em>
      
      
        2017
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5736944/" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>RNA microarrays and RNA-seq are nowadays standard technologies to study the transcriptional activity of cells. Most studies focus on tracking transcriptional changes caused by specific experimental conditions. Information referring to genes up- and downregulation is evaluated analyzing the behaviour of relatively large population of cells by averaging its properties. However, even assuming perfect sample homogeneity, different subpopulations of cells can exhibit diverse transcriptomic profiles, as they may follow different regulatory/signaling pathways. The purpose of this study is to provide a novel methodological scheme to account for possible internal, functional heterogeneity in homogeneous cell lines, including cancer ones. We propose a novel computational method to infer the proportion between subpopulations of cells that manifest various functional behaviour in a given sample. Our method was validated using two datasets from RNA microarray experiments. Both experiments aimed to examine cell viability in specific experimental conditions. The presented methodology can be easily extended to RNA-seq data as well as other molecular processes. Moreover, it complements standard tools to indicate most important networks from transcriptomic data and in particular could be useful in the analysis of cancer cell lines affected by biologically active compounds or drugs.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="8621194" class="col-sm-8">
    
      <div class="title">Renal cell carcinoma classification: a case study of pitfalls associated with metabolic landscape analysis</div>
      <div class="author">
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  Kostecki, M.,
                
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>In 2018 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</em>
      
      
        2018
      
      </div>
    

    <div class="links">
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
  </div>
</div>
</li></ol>
</div>

  </article>

</div>

    </div>

    <!-- Footer -->

    
<footer class="fixed-bottom">
  <div class="container mt-0">
    &copy; Copyright 2023 Igor  Ausiany.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Hosted by <a href="https://mimuw.edu.pl/" target="_blank">MIMUW</a>. Photos from <a href="https://unsplash.com" target="_blank">Unsplash</a>.

    
  </div>
</footer>



  </body>

  <!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>

  <!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>

  
<!-- Mansory & imagesLoaded -->
<script defer src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
<script defer src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
<script defer src="/~kgogolewski/igorteacher/assets/js/mansory.js" type="text/javascript"></script>


  


<!-- Load Common JS -->
<script src="/~kgogolewski/igorteacher/assets/js/common.js"></script>

<!-- Load DarkMode JS -->
<script src="/~kgogolewski/igorteacher/assets/js/dark_mode.js"></script>


</html>
