<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

<title>Igor  Ausiany | publications</title>
<meta name="description" content="Teaching portfolio.
">

<!-- Open Graph -->


<!-- Bootstrap & MDB -->
<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

<!-- Fonts & Icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"  integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">


<!-- Code Syntax Highlighting -->
<link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />

<!-- Styles -->
<link rel="shortcut icon" href="/~kgogolewski/igorteacher/assets/img/favicon.ico">
<link rel="stylesheet" href="/~kgogolewski/igorteacher/assets/css/main.css">

<link rel="canonical" href="/~kgogolewski/igorteacher/publications/">

<!-- Theming-->




    
<!-- MathJax -->
<script type="text/javascript">
  window.MathJax = {
    tex: {
      tags: 'ams'
    }
  };
</script>
<script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.1.2/es5/tex-mml-chtml.js"></script>
<script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>


  </head>

  <body class="fixed-top-nav ">

    <!-- Header -->

    <header>

    <!-- Nav Bar -->
    <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
    <div class="container">
      
      <a class="navbar-brand title font-weight-lighter" href="/~kgogolewski/igorteacher/">
       <span class="font-weight-bold">Igor</span>   Ausiany 
        
        <a class="support-button" href="https://zrzutka.pl/en/ghr8d9/pay">
            <i class="fas fa-piggy-bank"></i> Support Me
        </a>

       
      </a>
      
      <!-- Navbar Toogle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <!-- About -->
          <li class="nav-item ">
            <a class="nav-link" href="/~kgogolewski/igorteacher/">
              about
              
            </a>
          </li>
          
          <!-- Other pages -->
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/~kgogolewski/igorteacher/certificates/">
                certificates
                
              </a>
          </li>
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/~kgogolewski/igorteacher/projects/">
                projects
                
              </a>
          </li>
          
          
          
          <li class="nav-item active">
              <a class="nav-link" href="/~kgogolewski/igorteacher/publications/">
                publications
                
                <span class="sr-only">(current)</span>
                
              </a>
          </li>
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/~kgogolewski/igorteacher/teaching/">
                teaching
                
              </a>
          </li>
          
          
          
        </ul>
      </div>
    </div>
  </nav>

</header>


    <!-- Content -->

    <div class="container mt-5">
      <div class="post">

  <header class="post-header">
    <h2 class="post-title">publications</h1>
    <p class="post-description">below you can find a list of my publications in reversed chronological order.</p>
  </header>

  <article>
    <div class="publications">


  <h2 class="year">2022</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">BMC Genom</abbr>
    
  
  </div>

  <div id="Poszewiecka_Time_2022" class="col-sm-8">
    
      <div class="title">Revised time estimation of the ancestral human chromosome 2 fusion</div>
      <div class="author">
        
          
          
          
            
              
                
                  <a href="https://scholar.google.pl/citations?user=WlSEMowAAAAJ&amp;hl=en" target="_blank">Poszewiecka, B.</a>,
                
              
            
          
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  <a href="https://scholar.google.com/citations?user=lY9EfWwAAAAJ&amp;hl=en" target="_blank">Stankiewicz, P.</a>,
                
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>BMC Genomics</em>
      
      
        2022
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1186/s12864-022-08828-7" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Background:
The reduction of the chromosome number from 48 in the Great Apes to 46 in modern humans is thought to result from the end-to-end fusion of two ancestral non-human primate chromosomes forming the human chromosome 2 (HSA2). Genomic signatures of this event are the presence of inverted telomeric repeats at the HSA2 fusion site and a block of degenerate satellite sequences that mark the remnants of the ancestral centromere. It has been estimated that this fusion arose up to 4.5 million years ago (Mya).

Results:
We have developed an enhanced algorithm for the detection and efficient counting of the locally over-represented weak-to-strong (AT to GC) substitutions. By analyzing the enrichment of these substitutions around the fusion site of HSA2 we estimated its formation time at 0.9 Mya with a 95% confidence interval of 0.4-1.5 Mya. Additionally, based on the statistics derived from our algorithm, we have reconstructed the evolutionary distances among the Great Apes (Hominoidea).

Conclusions:
Our results shed light on the HSA2 fusion formation and provide a novel computational alternative for the estimation of the speciation chronology.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">NAR</abbr>
    
  
  </div>

  <div id="Poszewiecka_2022" class="col-sm-8">
    
      <div class="title">TADeus2: a web server facilitating the clinical diagnosis by pathogenicity assessment of structural variations disarranging 3D chromatin structure</div>
      <div class="author">
        
          
          
          
            
              
                
                  <a href="https://scholar.google.pl/citations?user=WlSEMowAAAAJ&amp;hl=en" target="_blank">Poszewiecka, B.</a>,
                
              
            
          
        
          
          
          
            
              
                
                  Pienkowski, V. M.,
                
              
            
          
        
          
          
          
            
              
                
                  Nowosad, K.,
                
              
            
          
        
          
          
          
            
              
                
                  Robin, J. D.,
                
              
            
          
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Nucleic Acids Research</em>
      
      
        2022
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://academic.oup.com/nar/article/50/W1/W744/6582161" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>In recent years great progress has been made in identification of structural variants (SV) in the human genome. However, the interpretation of SVs, especially located in non-coding DNA, remains challenging. One of the reasons stems in the lack of tools exclusively designed for clinical SVs evaluation acknowledging the 3D chromatin architecture. Therefore, we present TADeus2 a web server dedicated for a quick investigation of chromatin conformation changes, providing a visual framework for the interpretation of SVs affecting topologically associating domains (TADs). This tool provides a convenient visual inspection of SVs, both in a continuous genome view as well as from a rearrangement’s breakpoint perspective. Additionally, TADeus2 allows the user to assess the influence of analyzed SVs within flaking coding/non-coding regions based on the Hi-C matrix. Importantly, the SVs pathogenicity is quantified and ranked using TADA, ClassifyCNV tools and sampling-based P-value. TADeus2 is publicly available at https://tadeus2.mimuw.edu.pl.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Methods</abbr>
    
  
  </div>

  <div id="GOGOLEWSKI_CFR_2022" class="col-sm-8">
    
      <div class="title">Data-driven case fatality rate estimation for the primary lineage of SARS-CoV-2 in Poland</div>
      <div class="author">
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  Miasojedow, B.,
                
              
            
          
        
          
          
          
            
              
                
                  Sadkowska-Todys, M.,
                
              
            
          
        
          
          
          
            
              
                
                  Stepien, M.,
                
              
            
          
        
          
          
          
            
              
                
                  Demkow, U.,
                
              
            
          
        
          
          
          
            
              
                
                  Lech, A.,
                
              
            
          
        
          
          
          
            
              
                
                  <a href="https://scholar.google.pl/citations?user=hltmGf0AAAAJ&amp;hl=en" target="_blank">Szczurek, E.</a>,
                
              
            
          
        
          
          
          
            
              
                
                  Rabczenko, D.,
                
              
            
          
        
          
          
          
            
              
                
                  Rosinska, M,
                
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Methods</em>
      
      
        2022
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://www.sciencedirect.com/science/article/pii/S1046202322000123" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>After more than one and a half year since the COVID-19 pandemics outbreak the scientific world is constantly trying to understand its dynamics. In this paper of the case fatality rates (CFR) for COVID-19 we study the historic data regarding mortality in Poland during the first six months of pandemic, when no SARS-CoV-2 variants of concern were present among infected. To this end, we apply competing risk models to perform both uni- and multivariate analyses on specific subpopulations selected by different factors including the key indicators: age, sex, hospitalization. The study explores the case fatality rate to find out its decreasing trend in time. Furthermore, we describe the differences in mortality among hospitalized and other cases indicating a sudden increase of mortality among hospitalized cases at the end of the 2020 spring season. Exploratory and multivariate analysis revealed the real impact of each variable and besides the expected factors indicating increased mortality (age, comorbidities) we track more non-obvious indicators. Recent medical care as well as the identification of the source contact, independently of the comorbidities, significantly impact an individual mortality risk. As a result, the study provides a twofold insight into the COVID-19 mortality in Poland. On one hand we explore mortality in different groups with respect to different variables, on the other we indicate novel factors that may be crucial in reducing mortality. The later can be coped, e.g. by more efficient contact tracing and proper organization and management of the health care system to accompany those who need medical care independently of comorbidities or COVID-19 infection.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Comm Med</abbr>
    
  
  </div>

  <div id="Bracher2022" class="col-sm-8">
    
      <div class="title">National and subnational short-term forecasting of COVID-19 in Germany and Poland during early 2021</div>
      <div class="author">
        
          
          
          
            
              
                
                  Bracher, J.,
                
              
            
          
        
          
          
          
            
              
                
                  Wolffram, D.,
                
              
            
          
        
          
          
          
            
              
                
                  Deuschel, J.,
                
              
            
          
        
          
          
          
            
              
                
                  Görgen, K.,
                
              
            
          
        
          
          
          
            
              
                
                  Ketterer, J. L.,
                
              
            
          
        
          
          
          
            
              
                
                  Ullrich, A.,
                
              
            
          
        
          
          
          
            
              
                
                  Abbott, S.,
                
              
            
          
        
          
          
          
            
              
                
                  Barbarossa, M. V.,
                
              
            
          
        
          
          
          
            
              
                
                  Bertsimas, D.,
                
              
            
          
        
          
          
          
            
              
                
                  Bhatia, S.,
                
              
            
          
        
          
          
          
            
              
                
                  Bodych, M.,
                
              
            
          
        
          
          
          
            
              
                
                  Bosse, N. I.,
                
              
            
          
        
          
          
          
            
              
                
                  Burgard, J. P.,
                
              
            
          
        
          
          
          
            
              
                
                  Castro, L.,
                
              
            
          
        
          
          
          
            
              
                
                  Fairchild, G.,
                
              
            
          
        
          
          
          
            
              
                
                  Fiedler, J.,
                
              
            
          
        
          
          
          
            
              
                
                  Fuhrmann, J.,
                
              
            
          
        
          
          
          
            
              
                
                  Funk, S.,
                
              
            
          
        
          
          
          
            
              
                
                  <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>,
                
              
            
          
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  Heyder, S.,
                
              
            
          
        
          
          
          
            
              
                
                  Hotz, T.,
                
              
            
          
        
          
          
          
            
              
                
                  Kheifetz, Y.,
                
              
            
          
        
          
          
          
            
              
                
                  Kirsten, H.,
                
              
            
          
        
          
          
          
            
              
                
                  Krueger, T.,
                
              
            
          
        
          
          
          
            
              
                
                  Krymova, E.,
                
              
            
          
        
          
          
          
            
              
                
                  Leithäuser, N.,
                
              
            
          
        
          
          
          
            
              
                
                  Li, M. L.,
                
              
            
          
        
          
          
          
            
              
                
                  Meinke, J. H.,
                
              
            
          
        
          
          
          
            
              
                
                  Miasojedow, B.,
                
              
            
          
        
          
          
          
            
              
                
                  Michaud, I. J.,
                
              
            
          
        
          
          
          
            
              
                
                  Mohring, J.,
                
              
            
          
        
          
          
          
            
              
                
                  Nouvellet, P.,
                
              
            
          
        
          
          
          
            
              
                
                  Nowosielski, J. M.,
                
              
            
          
        
          
          
          
            
              
                
                  Ozanski, T.,
                
              
            
          
        
          
          
          
            
              
                
                  Radwan, M.,
                
              
            
          
        
          
          
          
            
              
                
                  Rakowski, F.,
                
              
            
          
        
          
          
          
            
              
                
                  Scholz, M.,
                
              
            
          
        
          
          
          
            
              
                
                  Soni, S.,
                
              
            
          
        
          
          
          
            
              
                
                  Srivastava, A.,
                
              
            
          
        
          
          
          
            
              
                
                  Gneiting, T.,
                
              
            
          
        
          
          
          
            
              
                
                  and Schienle, M.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Communications Medicine</em>
      
      
        2022
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1038/s43856-022-00191-8" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Background:
During the COVID-19 pandemic there has been a strong interest in forecasts of the short-term development of epidemiological indicators to inform decision makers. In this study we evaluate probabilistic real-time predictions of confirmed cases and deaths from COVID-19 in Germany and Poland for the period from January through April 2021.

Methods:
We evaluate probabilistic real-time predictions of confirmed cases and deaths from COVID-19 in Germany and Poland. These were issued by 15 different forecasting models, run by independent research teams. Moreover, we study the performance of combined ensemble forecasts. Evaluation of probabilistic forecasts is based on proper scoring rules, along with interval coverage proportions to assess calibration. The presented work is part of a pre-registered evaluation study.

Results:
We find that many, though not all, models outperform a simple baseline model up to four weeks ahead for the considered targets. Ensemble methods show very good relative performance. The addressed time period is characterized by rather stable non-pharmaceutical interventions in both countries, making short-term predictions more straightforward than in previous periods. However, major trend changes in reported cases, like the rebound in cases due to the rise of the B.1.1.7 (Alpha) variant in March 2021, prove challenging to predict.

Conclusions:
Multi-model approaches can help to improve the performance of epidemiological forecasts. However, while death numbers can be predicted with some success based on current case and hospitalization data, predictability of case numbers remains low beyond quite short time horizons. Additional data sources including sequencing and mobility data, which were not extensively used in the present study, may help to improve performance.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Comm Med</abbr>
    
  
  </div>

  <div id="Krueger2022" class="col-sm-8">
    
      <div class="title">Risk assessment of COVID-19 epidemic resurgence in relation to SARS-CoV-2 variants and vaccination passes</div>
      <div class="author">
        
          
          
          
            
              
                
                  Krueger, T.,
                
              
            
          
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  Bodych, M.,
                
              
            
          
        
          
          
          
            
              
                
                  <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>,
                
              
            
          
        
          
          
          
            
              
                
                  Giordano, G.,
                
              
            
          
        
          
          
          
            
              
                
                  Cuschieri, S.,
                
              
            
          
        
          
          
          
            
              
                
                  Czypionka, T.,
                
              
            
          
        
          
          
          
            
              
                
                  Perc, M.,
                
              
            
          
        
          
          
          
            
              
                
                  Petelos, E.,
                
              
            
          
        
          
          
          
            
              
                
                  Rosińska, M.,
                
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.pl/citations?user=hltmGf0AAAAJ&amp;hl=en" target="_blank">Szczurek, E.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Communications Medicine</em>
      
      
        2022
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://www.nature.com/articles/s43856-022-00084-w" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The introduction of COVID-19 vaccination passes (VPs) by many countries coincided with the Delta variant fast becoming dominant across Europe. A thorough assessment of their impact on epidemic dynamics is still lacking. Here, we propose the VAP-SIRS model that considers possibly lower restrictions for the VP holders than for the rest of the population, imperfect vaccination effectiveness against infection, rates of (re-)vaccination and waning immunity, fraction of never-vaccinated, and the increased transmissibility of the Delta variant. Some predicted epidemic scenarios for realistic parameter values yield new COVID-19 infection waves within two years, and high daily case numbers in the endemic state, even without introducing VPs and granting more freedom to their holders. Still, suitable adaptive policies can avoid unfavorable outcomes. While VP holders could initially be allowed more freedom, the lack of full vaccine effectiveness and increased transmissibility will require accelerated (re-)vaccination, wide-spread immunity surveillance, and/or minimal long-term common restrictions.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2021</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Sci Rep</abbr>
    
  
  </div>

  <div id="MakarovaSciRep" class="col-sm-8">
    
      <div class="title">Harvest time affects antioxidant capacity, total polyphenol and flavonoid content of Polish St John’s wort’s (Hypericum perforatum L.) flowers</div>
      <div class="author">
        
          
          
          
            
              
                
                  Makarova, K.,
                
              
            
          
        
          
          
          
            
              
                
                  Sajkowska-Kozielewicz, J. J.,
                
              
            
          
        
          
          
          
            
              
                
                  Zawada, K.,
                
              
            
          
        
          
          
          
            
              
                
                  Olchowik-Grabarek, E.,
                
              
            
          
        
          
          
          
            
              
                
                  Ciach, M. A.,
                
              
            
          
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  Dobros, N.,
                
              
            
          
        
          
          
          
            
              
                
                  Ciechowicz, P.,
                
              
            
          
        
          
          
          
            
              
                
                  Freichels, H.,
                
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Scientific Reports</em>
      
      
        2021
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://www.nature.com/articles/s41598-021-83409-4" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
      
      <a href="/~kgogolewski/igorteacher/assets/pdf/scientificReports2021" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
      
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The polyphenol content and antioxidant capacity of hyperforin and hypericin-standardized H. perforatum L. extracts may vary due to the harvest time. In this work, ethanol and ethanol-water extracts of air-dried and lyophilized flowers of H. perforatum L., collected throughout a vegetation season in central Poland, were studied. Air-dried flowers extracts had higher polyphenol (371 mg GAE/g) and flavonoid (160 mg CAE/g) content, DPPH radical scavenging (1672 mg DPPH/g), ORAC (5214 µmol TE/g) and FRAP (2.54 mmol Fe2+/g) than lyophilized flowers extracts (238 mg GAE/g, 107 mg CAE/g, 1287 mg DPPH/g, 3313 µmol TE/g and 0.31 mmol Fe2+/g, respectively). Principal component analysis showed that the collection date influenced the flavonoid and polyphenol contents and FRAP of ethanol extracts, and DPPH and ORAC values of ethanol-water extracts. The ethanol extracts with the highest polyphenol and flavonoid content protected human erythrocytes against bisphenol A-induced damage. Both high field and benchtop NMR spectra of selected extracts, revealed differences in composition caused by extraction solvent and raw material collection date. Moreover, we have shown that benchtop NMR can be used to detect the compositional variation of extracts if the assignment of signals is done previously.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Nat Comm</abbr>
    
  
  </div>

  <div id="Bracher2021" class="col-sm-8">
    
      <div class="title">A pre-registered short-term forecasting study of COVID-19 in Germany and Poland during the second wave</div>
      <div class="author">
        
          
          
          
            
              
                
                  Bracher, J.,
                
              
            
          
        
          
          
          
            
              
                
                  Wolffram, D.,
                
              
            
          
        
          
          
          
            
              
                
                  Deuschel, J.,
                
              
            
          
        
          
          
          
            
              
                
                  Görgen, K.,
                
              
            
          
        
          
          
          
            
              
                
                  Ketterer, J. L.,
                
              
            
          
        
          
          
          
            
              
                
                  Ullrich, A.,
                
              
            
          
        
          
          
          
            
              
                
                  Abbott, S.,
                
              
            
          
        
          
          
          
            
              
                
                  Barbarossa, M. V.,
                
              
            
          
        
          
          
          
            
              
                
                  Bertsimas, D.,
                
              
            
          
        
          
          
          
            
              
                
                  Bhatia, S.,
                
              
            
          
        
          
          
          
            
              
                
                  Bodych, M.,
                
              
            
          
        
          
          
          
            
              
                
                  Bosse, N. I.,
                
              
            
          
        
          
          
          
            
              
                
                  Burgard, J. P.,
                
              
            
          
        
          
          
          
            
              
                
                  Castro, L.,
                
              
            
          
        
          
          
          
            
              
                
                  Fairchild, G.,
                
              
            
          
        
          
          
          
            
              
                
                  Fuhrmann, J.,
                
              
            
          
        
          
          
          
            
              
                
                  Funk, S.,
                
              
            
          
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  Gu, Q.,
                
              
            
          
        
          
          
          
            
              
                
                  Heyder, S.,
                
              
            
          
        
          
          
          
            
              
                
                  Hotz, T.,
                
              
            
          
        
          
          
          
            
              
                
                  Kheifetz, Y.,
                
              
            
          
        
          
          
          
            
              
                
                  Kirsten, H.,
                
              
            
          
        
          
          
          
            
              
                
                  Krueger, T.,
                
              
            
          
        
          
          
          
            
              
                
                  Krymova, E.,
                
              
            
          
        
          
          
          
            
              
                
                  Li, M. L.,
                
              
            
          
        
          
          
          
            
              
                
                  Meinke, J. H.,
                
              
            
          
        
          
          
          
            
              
                
                  Michaud, I. J.,
                
              
            
          
        
          
          
          
            
              
                
                  Niedzielewski, K.,
                
              
            
          
        
          
          
          
            
              
                
                  Ożański, T.,
                
              
            
          
        
          
          
          
            
              
                
                  Rakowski, F.,
                
              
            
          
        
          
          
          
            
              
                
                  Scholz, M.,
                
              
            
          
        
          
          
          
            
              
                
                  Soni, S.,
                
              
            
          
        
          
          
          
            
              
                
                  Srivastava, A.,
                
              
            
          
        
          
          
          
            
              
                
                  Zieliński, J.,
                
              
            
          
        
          
          
          
            
              
                
                  Zou, D.,
                
              
            
          
        
          
          
          
            
              
                
                  Gneiting, T.,
                
              
            
          
        
          
          
          
            
              
                
                  and Schienle, M.
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Nature Communications</em>
      
      
        2021
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1038/s41467-021-25207-0" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Disease modelling has had considerable policy impact during the ongoing COVID-19 pandemic, and it is increasingly acknowledged that combining multiple models can improve the reliability of outputs. Here we report insights from ten weeks of collaborative short-term forecasting of COVID-19 in Germany and Poland. The study period covers the onset of the second wave in both countries, with tightening non-pharmaceutical interventions (NPIs) and subsequently a decay (Poland) or plateau and renewed increase (Germany) in reported cases. Thirteen independent teams provided probabilistic real-time forecasts of COVID-19 cases and deaths. These were reported for lead times of one to four weeks, with evaluation focused on one- and two-week horizons, which are less affected by changing NPIs. Heterogeneity between forecasts was considerable both in terms of point predictions and forecast spread. Ensemble forecasts showed good relative performance, in particular in terms of coverage, but did not clearly dominate single-model predictions. The study was preregistered and will be followed up in future phases of the pandemic.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2020</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">Entropy</abbr>
    
  
  </div>

  <div id="GogolewskiEntropy" class="col-sm-8">
    
      <div class="title">Low Entropy Sub-Networks Prevent the Integration of Metabolomic and Transcriptomic Data</div>
      <div class="author">
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  Kostecki, M.,
                
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Entropy (Basel)</em>
      
      
        2020
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://www.mdpi.com/1099-4300/22/11/1238" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
      
      <a href="/~kgogolewski/igorteacher/assets/pdf/gogolewski2020Entropy.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a>
      
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The constantly and rapidly increasing amount of the biological data gained from many different high-throughput experiments opens up new possibilities for data- and model-driven inference. Yet, alongside, emerges a problem of risks related to data integration techniques. The latter are not so widely taken account of. Especially, the approaches based on the flux balance analysis (FBA) are sensitive to the structure of a metabolic network for which the low-entropy clusters can prevent the inference from the activity of the metabolic reactions. In the following article, we set forth problems that may arise during the integration of metabolomic data with gene expression datasets. We analyze common pitfalls, provide their possible solutions, and exemplify them by a case study of the renal cell carcinoma (RCC). Using the proposed approach we provide a metabolic description of the known morphological RCC subtypes and suggest a possible existence of the poor-prognosis cluster of patients, which are commonly characterized by the low activity of the drug transporting enzymes crucial in the chemotherapy. This discovery suits and extends the already known poor-prognosis characteristics of RCC. Finally, the goal of this work is also to point out the problem that arises from the integration of high-throughput data with the inherently nonuniform, manually curated low-throughput data. In such cases, the over-represented information may potentially overshadow the non-trivial discoveries.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2019</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">JCB</abbr>
    
  
  </div>

  <div id="GogolewskiJCB" class="col-sm-8">
    
      <div class="title">Truncated Robust Principal Component Analysis and Noise Reduction for Single Cell RNA Sequencing Data</div>
      <div class="author">
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  <a href="https://scholar.google.pl/citations?user=QGnkxbQAAAAJ&amp;hl=en" target="_blank">Sykulski, M.</a>,
                
              
            
          
        
          
          
          
            
              
                
                  Chung, N. C.,
                
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>J Comput Biol</em>
      
      
        2019
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1089/cmb.2018.0255" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The development of single cell RNA sequencing (scRNA-seq) has enabled innovative approaches to investigating mRNA abundances. In our study, we are interested in extracting the systematic patterns of scRNA-seq data in an unsupervised manner; thus, we have developed two extensions of robust principal component analysis (RPCA). First, we present a truncated version of RPCA (tRPCA), which is much faster and memory efficient. Second, we introduce a noise reduction in tRPCA with L2 regularization. Unlike RPCA that only considers a low-rank L and sparse S matrices, the proposed method can also extract a noise E matrix inherent in modern genomic data. We demonstrate its usefulness by applying our methods on the peripheral blood mononuclear cell scRNA-seq data. Particularly, the clustering of a low-rank L matrix showcases better classification of unlabeled single cells. Overall, the proposed variants are well suited for high-dimensional and noisy data that are routinely generated in genomics.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2018</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="8621194" class="col-sm-8">
    
      <div class="title">Renal cell carcinoma classification: a case study of pitfalls associated with metabolic landscape analysis</div>
      <div class="author">
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  Kostecki, M.,
                
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>In 2018 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</em>
      
      
        2018
      
      </div>
    

    <div class="links">
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="8542112" class="col-sm-8">
    
      <div class="title">PCA-like Methods for the Integration of Single Cell RNA-seq Data with Metabolic Networks</div>
      <div class="author">
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>In 2018 IEEE 8th International Conference on Computational Advances in Bio and Medical Sciences (ICCABS)</em>
      
      
        2018
      
      </div>
    

    <div class="links">
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="10.1007/978-3-319-94968-0_32" class="col-sm-8">
    
      <div class="title">Truncated Robust Principal Component Analysis and Noise Reduction for Single Cell RNA-seq Data</div>
      <div class="author">
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  <a href="https://scholar.google.pl/citations?user=QGnkxbQAAAAJ&amp;hl=en" target="_blank">Sykulski, M.</a>,
                
              
            
          
        
          
          
          
            
              
                
                  Chung, N.C.,
                
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>In Bioinformatics Research and Applications</em>
      
      
        2018
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The development of single cell RNA sequencing (scRNA-seq) has enabled innovative approaches to investigating mRNA abundances. In our study, we are interested in extracting the systematic patterns of scRNA-seq data in an unsupervised manner, thus we have developed two extensions of robust principal component analysis (RPCA). First, we present a truncated version of RPCA (tRPCA), that is much faster and memory efficient. Second, we introduce a noise reduction in tRPCA with }}L_2}} regularization (tRPCAL2). Unlike RPCA that only considers a low-rank L and sparse S matrices, the proposed method can also extract a noise E matrix inherent in modern genomic data. We demonstrate its usefulness by applying our methods on the peripheral blood mononuclear cell (PBMC) scRNA-seq data. Particularly, the clustering of a low-rank L matrix showcases better classification of unlabeled single cells. Overall, the proposed variants are well-suited for high-dimensional and noisy data that are routinely generated in genomics.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2017</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">BioMedRes</abbr>
    
  
  </div>

  <div id="GogolewskiBioMedRes" class="col-sm-8">
    
      <div class="title">Inferring Molecular Processes Heterogeneity from Transcriptional Data</div>
      <div class="author">
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  Wronowska, W.,
                
              
            
          
        
          
          
          
            
              
                
                  Lech, A.,
                
              
            
          
        
          
          
          
            
              
                
                  Lesyng, B.,
                
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Biomed Res Int</em>
      
      
        2017
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5736944/" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>RNA microarrays and RNA-seq are nowadays standard technologies to study the transcriptional activity of cells. Most studies focus on tracking transcriptional changes caused by specific experimental conditions. Information referring to genes up- and downregulation is evaluated analyzing the behaviour of relatively large population of cells by averaging its properties. However, even assuming perfect sample homogeneity, different subpopulations of cells can exhibit diverse transcriptomic profiles, as they may follow different regulatory/signaling pathways. The purpose of this study is to provide a novel methodological scheme to account for possible internal, functional heterogeneity in homogeneous cell lines, including cancer ones. We propose a novel computational method to infer the proportion between subpopulations of cells that manifest various functional behaviour in a given sample. Our method was validated using two datasets from RNA microarray experiments. Both experiments aimed to examine cell viability in specific experimental conditions. The presented methodology can be easily extended to RNA-seq data as well as other molecular processes. Moreover, it complements standard tools to indicate most important networks from transcriptomic data and in particular could be useful in the analysis of cancer cell lines affected by biologically active compounds or drugs.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2016</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">BiolOpen</abbr>
    
  
  </div>

  <div id="DharmadhikariBiolOpen" class="col-sm-8">
    
      <div class="title">Lethal lung hypoplasia and vascular defects in mice with conditional Foxf1 overexpression</div>
      <div class="author">
        
          
          
          
            
              
                
                  Dharmadhikari, A. V.,
                
              
            
          
        
          
          
          
            
              
                
                  Sun, J. J.,
                
              
            
          
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  Carofino, B. L.,
                
              
            
          
        
          
          
          
            
              
                
                  Ustiyan, V.,
                
              
            
          
        
          
          
          
            
              
                
                  Hill, M.,
                
              
            
          
        
          
          
          
            
              
                
                  Majewski, T.,
                
              
            
          
        
          
          
          
            
              
                
                  Szafranski, P.,
                
              
            
          
        
          
          
          
            
              
                
                  Justice, M. J.,
                
              
            
          
        
          
          
          
            
              
                
                  Ray, R. S.,
                
              
            
          
        
          
          
          
            
              
                
                  Dickinson, M. E.,
                
              
            
          
        
          
          
          
            
              
                
                  Kalinichenko, V. V.,
                
              
            
          
        
          
          
          
            
              
                
                  <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>,
                
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=lY9EfWwAAAAJ&amp;hl=en" target="_blank">Stankiewicz, P.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>Biology Open</em>
      
      
        2016
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5155529/" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>FOXF1 heterozygous point mutations and genomic deletions have been reported in newborns with the neonatally lethal lung developmental disorder, alveolar capillary dysplasia with misalignment of pulmonary veins (ACDMPV). However, no gain-of-function mutations in FOXF1 have been identified yet in any human disease conditions. To study the effects of FOXF1 overexpression in lung development, we generated a Foxf1 overexpression mouse model by knocking-in a Cre-inducible Foxf1 allele into the ROSA26 (R26) locus. The mice were phenotyped using micro-computed tomography (micro-CT), head-out plethysmography, ChIP-seq and transcriptome analyses, immunohistochemistry, and lung histopathology. Thirty-five percent of heterozygous R26-Lox-Stop-Lox (LSL)-Foxf1 embryonic day (E)15.5 embryos exhibit subcutaneous edema, hemorrhages and die perinatally when bred to Tie2-cre mice, which targets Foxf1 overexpression to endothelial and hematopoietic cells. Histopathological and micro-CT evaluations revealed that R26Foxf1; Tie2-cre embryos have immature lungs with a diminished vascular network. Neonates exhibited respiratory deficits verified by detailed plethysmography studies. ChIP-seq and transcriptome analyses in E18.5 lungs identified Sox11, Ghr, Ednrb, and Slit2 as potential downstream targets of FOXF1. Our study shows that overexpression of the highly dosage-sensitive Foxf1 impairs lung development and causes vascular abnormalities. This has important clinical implications when considering potential gene therapy approaches to treat disorders of FOXF1 abnormal dosage, such as ACDMPV.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2014</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
    
    <abbr class="badge">BMCMedGen</abbr>
    
  
  </div>

  <div id="DharmadhikariBMCMedGen" class="col-sm-8">
    
      <div class="title">Molecular and clinical analyses of 16q24.1 duplications involving FOXF1 identify an evolutionarily unstable large minisatellite</div>
      <div class="author">
        
          
          
          
            
              
                
                  Dharmadhikari, A. V.,
                
              
            
          
        
          
          
          
            
              
                
                  <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, T.</a>,
                
              
            
          
        
          
          
          
            
              
                
                  Szafranski, P.,
                
              
            
          
        
          
          
          
            
              
                
                  Cao, W.,
                
              
            
          
        
          
          
          
            
              
                
                  Probst, F. J.,
                
              
            
          
        
          
          
          
            
              
                
                  Jin, W.,
                
              
            
          
        
          
          
          
            
              
                
                  Fang, P.,
                
              
            
          
        
          
          
          
            
              
                <em>Gogolewski, K.</em>,
              
            
          
        
          
          
          
            
              
                
                  <a href="https://scholar.google.com/citations?user=ZlE1X8IAAAAJ&amp;hl=en" target="_blank">Gambin, A.</a>,
                
              
            
          
        
          
          
          
            
              
                
                  George-Abraham, J. K.,
                
              
            
          
        
          
          
          
            
              
                
                  Golla, S.,
                
              
            
          
        
          
          
          
            
              
                
                  Boidein, F.,
                
              
            
          
        
          
          
          
            
              
                
                  Duban-Bedu, B.,
                
              
            
          
        
          
          
          
            
              
                
                  Delobel, B.,
                
              
            
          
        
          
          
          
            
              
                
                  Andrieux, J.,
                
              
            
          
        
          
          
          
            
              
                
                  Becker, K.,
                
              
            
          
        
          
          
          
            
              
                
                  Holinski-Feder, E.,
                
              
            
          
        
          
          
          
            
              
                
                  Cheung, S. W.,
                
              
            
          
        
          
          
          
            
              
                
                  and <a href="https://scholar.google.com/citations?user=lY9EfWwAAAAJ&amp;hl=en" target="_blank">Stankiewicz, P.</a>
                
              
            
          
        
      </div>

      <div class="periodical">
      
        <em>BMC Med Genet</em>
      
      
        2014
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://pubmed.ncbi.nlm.nih.gov/25472632/" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Background: Point mutations or genomic deletions of FOXF1 result in a lethal developmental lung disease Alveolar Capillary Dysplasia with Misalignment of Pulmonary Veins. However, the clinical consequences of the constitutively increased dosage of FOXF1 are unknown.
Methods: Copy-number variations and their parental origin were identified using a combination of array CGH, long-range PCR, DNA sequencing, and microsatellite analyses. Minisatellite sequences across different species were compared using a gready clustering algorithm and genome-wide analysis of the distribution of minisatellite sequences was performed using R statistical software.
Results: We report four unrelated families with 16q24.1 duplications encompassing entire FOXF1. In a 4-year-old boy with speech delay and a café-au-lait macule, we identified an  15 kb 16q24.1 duplication inherited from the reportedly healthy father, in addition to a de novo  1.09 Mb mosaic 17q11.2 NF1 deletion. In a 13-year-old patient with autism and mood disorder, we found an  0.3 Mb duplication harboring FOXF1 and an  0.5 Mb 16q23.3 duplication, both inherited from the father with bipolar disorder. In a 47-year old patient with pyloric stenosis, mesenterium commune, and aplasia of the appendix, we identified an  0.4 Mb duplication in 16q24.1 encompassing 16 genes including FOXF1. The patient transmitted the duplication to her daughter, who presented with similar symptoms. In a fourth patient with speech and motor delay, and borderline intellectual disability, we identified an  1.7 Mb FOXF1 duplication adjacent to a large minisatellite. This duplication has a complex structure and arose de novo on the maternal chromosome, likely as a result of a DNA replication error initiated by the adjacent large tandem repeat. Using bioinformatic and array CGH analyses of the minisatellite, we found a large variation of its size in several different species and individuals, demonstrating both its evolutionarily instability and population polymorphism.
Conclusions: Our data indicate that constitutional duplication of FOXF1 in humans is not associated with any pediatric lung abnormalities. We propose that patients with gut malrotation, pyloric or duodenal stenosis, and gall bladder agenesis should be tested for FOXF1 alterations. We suggest that instability of minisatellites greater than 1 kb can lead to structural variation due to DNA replication errors.</p>
    </div>
    
  </div>
</div>
</li></ol>


</div>


  </article>

</div>

    </div>

    <!-- Footer -->

    
<footer class="fixed-bottom">
  <div class="container mt-0">
    &copy; Copyright 2023 Igor  Ausiany.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Hosted by <a href="https://mimuw.edu.pl/" target="_blank">MIMUW</a>. Photos from <a href="https://unsplash.com" target="_blank">Unsplash</a>.

    
  </div>
</footer>



  </body>

  <!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>

  <!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>

  
<!-- Mansory & imagesLoaded -->
<script defer src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
<script defer src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
<script defer src="/~kgogolewski/igorteacher/assets/js/mansory.js" type="text/javascript"></script>


  


<!-- Load Common JS -->
<script src="/~kgogolewski/igorteacher/assets/js/common.js"></script>

<!-- Load DarkMode JS -->
<script src="/~kgogolewski/igorteacher/assets/js/dark_mode.js"></script>


</html>
